var/spack/repos/builtin/packages/py-pennylane-lightning/package.py

```diff
@@ -17,7 +17,16 @@ class PyPennylaneLightning(CMakePackage, PythonExtension):
     maintainers("mlxd", "AmintorDusko")
 
     version("master", branch="master")
-    version("0.29.0", sha256="da9912f0286d1a54051cc19cf8bdbdcd732795636274c95f376db72a88e52d85")
+    version(
+        "0.29.0",
+        sha256="da9912f0286d1a54051cc19cf8bdbdcd732795636274c95f376db72a88e52d85",
+        preferred=True,
```

Comment:
The most recent stable release is already preferred by default. You can't prefer multiple versions.

Reply:
Good, I'll remove that. For some reason, my `spack` would select `master` even after uninstalling everything and reconcretizing.